packaging/firehol/firehol.functions

```diff
@@ -11,12 +11,12 @@ firehol_check_file() {
     sbin/Makefile.in)
       :
     ;;
-    configure.ac|sbin/commands.sed.in)
-      check_commands sbin/firehol.in || status=1
-      check_commands sbin/fireqos.in || status=1
-      check_commands sbin/link-balancer.in || status=1
-      check_commands sbin/update-ipsets.in || status=1
-      check_commands sbin/vnetbuild.in || status=1
+    configure.ac)
+      check_commands sbin/firehol.in.in || status=1
+      check_commands sbin/fireqos.in.in || status=1
+      check_commands sbin/link-balancer.in.in || status=1
+      check_commands sbin/update-ipsets.in.in || status=1
+      check_commands sbin/vnetbuild.in.in || status=1
```

Comment:
This function is called as part of commit hooks. It is checking e.g. that if `@LS@` is used in a script that it does not get removed from `configure.ac` and other such things. In the case where `configure.ac` is changed we check all the scripts, just to be sure.